app/driver.f90

```diff
@@ -116,12 +118,30 @@ subroutine run_main(config, error)
       call wrap_to_central_cell(mol%xyz, mol%lattice, mol%periodic)
    end if
 
+   ga = config%ga
+   gc = config%gc
    if (config%mbdscale) s9 = config%inp%s9
    if (config%rational) then
       if (config%has_param) then
          param = config%inp
       else
-         call get_rational_damping(config%method, param, s9)
+         is = index(config%method, '/')
+         if (is == 0) is = len_trim(config%method) + 1
+         functional = lowercase(config%method(:is-1))
+         id = get_functional_id(functional)
+
+         ! special case: r2SCAN-3c (modifies s9, ga, gc)
+         if (id == 102) then
```

Comment:
I would either make the r2SCAN-3c ID public or define a function to perform this check in the param module.

Reply:
I made the r2SCAN-3c ID public.